ciao-4.11/contrib/bin/download_obsid_caldb

```diff
@@ -467,6 +461,8 @@ def retrieve_files_by_tool( ftp, rootdir, tool, infile, outdir, clobber, missing
 
         products = cc.search
         retrieve_caldb_file( ftp, rootdir, products, outdir, clobber, missing )
+
+        # What does the following line do?
```

Comment:
At the C/C++ level it free memory/etc.  Presumably something similar in python.  

[The whole caldb module is not very pythonic.]

Reply:
But if it's a function then shouldn't it be `cc.close()`? Then again you might say that about `cc.search` so all bets are off here, as you say.

From my limited testing the `close` attribute seemed to be `None` (if memory serves).

Reply:
Replaced my comment with another comment for when I next look at this...